cibuildwheel/action.yml

```diff
@@ -83,16 +87,25 @@ runs:
         skbuild_config_opts="${{ inputs.skbuild-configure-options}}"
         skbuild_config_opts="${skbuild_config_opts} -DCMAKE_C_COMPILER_LAUNCHER=/usr/bin/sccache -DCMAKE_CXX_COMPILER_LAUNCHER=/usr/bin/sccache -DCMAKE_CUDA_COMPILER_LAUNCHER=/usr/bin/sccache"
 
-        # use gha-tools rapids-pip-wheel-version to generate wheel version
-        versioneer_override="$(rapids-pip-wheel-version ${{ inputs.epoch-timestamp }})"
-
         # convert 3.x to cp3x-*
         python_ver=${{ inputs.python-version }}
         python_ver="${python_ver//./}"
         echo "CIBW_BUILD=cp${python_ver}-*" >> "${GITHUB_ENV}"
 
+        # Use gha-tools rapids-pip-wheel-version to generate wheel version then update the necessary files
+        versioneer_override="$(rapids-pip-wheel-version ${{ inputs.epoch-timestamp }})"
+        env_var="CIBW_ENVIRONMENT=SKBUILD_CONFIGURE_OPTIONS='${skbuild_config_opts}' SKBUILD_BUILD_OPTIONS='${{ inputs.skbuild-build-options }}' ${{ inputs.cibw-environment }} RAPIDS_PY_WHEEL_CUDA_SUFFIX='${{ inputs.cuda-suffix }}' SCCACHE_S3_KEY_PREFIX=gha-cibw SCCACHE_REGION=us-west-2 SCCACHE_IDLE_TIMEOUT=32768 SCCACHE_BUCKET=rapids-sccache SCCACHE_S3_USE_SSL=true AWS_ACCESS_KEY_ID=${{ inputs.sccache-aws-access-key-id }} AWS_SECRET_ACCESS_KEY=${{ inputs.sccache-aws-secret-access-key }} PIP_INDEX_URL=https://pypi.k8s.rapids.ai/simple PIP_FIND_LINKS=/project/local-wheelhouse" >> "${GITHUB_ENV}"
+
+        if [ "${{ inputs.uses-versioneer }}" = true ]; then
+          env_var="${env_var} RAPIDS_PY_WHEEL_VERSIONEER_OVERRIDE='${versioneer_override}'"
+        else
+          bash ci/release/apply_wheel_modifications.sh ${versioneer_override}
+          echo "The git diff after modifying the versions is:"
+          git diff
```

Comment:
In the PR you mention "ci/release/modify_wheel_build_version.sh" but this is "apply_wheel_modifications.sh"; are both scripts present or did you intend one over the other?

Reply:
I changed the name after creating this PR because I realized that I intended to do more than just version modification in downstream work. I'll update the PR description.